composer.json

```diff
@@ -67,6 +67,12 @@
 		"check-cs": [
 			"@php ./vendor/squizlabs/php_codesniffer/scripts/phpcs"
 		],
+		"premium-check-cs": [
+			"composer require yoast/yoastcs:~1.2.1 --update-with-dependencies --no-suggest --no-interaction",
+			"@php ./vendor/squizlabs/php_codesniffer/scripts/phpcs --ignore=*/premium/cli/ --runtime-set ignore_warnings_on_exit 1",
```

Comment:
`/scripts/` should be `/bin/`
```suggestion
			"@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --ignore=*/premium/cli/ --runtime-set ignore_warnings_on_exit 1",
```

Reply:
Oops... good catch! I will fix up the commit (once I can get past a git remote delta which is corrupted)

Reply:
Got past the corrupted git issue and have fixed this up now.